.github/workflows/ci.yml

```
@@ -0,0 +1,246 @@
+      - name: Upload Documentation
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: documentation
+          path: |
+            **/build/dokka/
+
+  security-scan:
+    name: Security Scan
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)